class.jetpack-options.php

```diff
@@ -120,6 +120,24 @@ public static function get_option( $name, $default = false ) {
 		return $default;
 	}
 
+	/**
+	 * Returns the requested option, and ensures it's autoloaded in the future.
+	 * This does _not_ adjust the prefix in any way (does not prefix jetpack_%)
+	 *
+	 * @param string $name Option name
+	 * @param mixed $default (optional)
+	 */
+	public static function get_option_and_ensure_autoload( $name, $default ) {
+		$value = get_option( $name );
+		
+		if ( $value === false ) {
+			update_option( $name, $default, null, true );
```

Comment:
Since `update_option` does nothing if $default is false, there's no harm in calling it since it doesn't hit the DB - however I can check if `$default === false` in the if-statement above it juuuust in case.

In all the places where we call this function, I call it with a default of '0' if I want to store a falsy value.